### PR TITLE
chore: New UX Consistency TRG DRAFT

### PR DESCRIPTION
**Pull request by kelaja**
This TRG DRAFT can be used for defining the UX consistency tractus-x release guidelines (TRG)

### CODE REVIEW

**Comment by kelaja** [comment]
Perspectively all Catena-X specific Intellectual Property need to be removed from tractus-x release guidelines.
One suggestion is to exchange the Catena-X logo with tracts-x logo.
Links to closed systems like figma need to be removed due to missing licence, open source alternative should be found.

**Comment by danielmiehle** [comment]
@kelaja @jjeroch : is this PR still relevant or can we close it?

**Review by kelaja — COMMENTED** [review]
chore: TRG Draft for UX Guidelines

**Review by kelaja — COMMENTED** [review]
chore: new UX TRG

File: docs/release/trg-0/trg-1-09.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: TRG 1.09 - UX consistency Style Guideline for UI
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:2]: UX consistency / Styleguide

File: docs/release/trg-0/trg-1-09.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+|-------------------------------------------|-----------------|----------------------------------------------------|
+| Draft - MISSING CLOSED INFORMATION      | 10-Jan-2024     | Transfered from Release Acceptance Criteria into TRG|
+
+## Why
+
+Ensuring UX consistency through a style guideline is essential for a unified and user-friendly experience across front end modules.
+
+## Description
+
+- UX consistency is mandatory for FrontEnd modules.
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:20]: ...is essential...

File: docs/release/trg-0/trg-1-09.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- User Interfaces should align with the Catena-X Style Guidelines.
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:21]: ...therefore should adhere to the Tractus-X style guidelines.

File: docs/release/trg-0/trg-1-09.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+> User interface style review has been executed.
+> Review feedback (if existing) got incorporated.
+> All findings are assessed.
+> All findings (high/very high) are fixed or cleaned up (evidence by re-review).
+> Approval of the application CX Style conformity is available (given by the review owner).
+
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:28]: delete line items 23 to 27

File: docs/release/trg-0/trg-1-09.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+## Best Practice
+
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:30]: Make use of the public available shared component library (react supported) to easily develop applications that are in line with the Tractus-X style guidelines.

File: docs/release/trg-0/trg-1-09.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Obtain approval from the Style Guideline Owner, prior to Gate review.
+- Get in contact with Style Guideline Owner and expect a review loop.
+
+Style Components:
+
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:35]: consolidate and update all relevant LINKs

File: docs/release/trg-0/trg-1-09.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- LINK to Style Guideline
+- LINK to FrontEnd validations (will be added asap)
+
+> EXTERNAL LINKS
+> CX references
+> Offical style guide accessible on Catena-X website
+
+Please note, you can use the official public available CX shared component library (react supported) to easily develop applications that are in-line with the CX style guidelines.
```
Comment (RolaH1t) [inline docs/release/trg-0/trg-1-09.md:43]: delete line item 43

**Comment by RolaH1t** [comment]
Pls help to wrap this up and officially utilize the new TRG for Release 24.05

File: docs/release/trg-0/trg-1-09.md (hunk 1 above)
Comment (FaGru3n) [inline docs/release/trg-0/trg-1-09.md:2]: Hi @RolaH1t  you can directly share a suggestion for the code within this pr like: 
```suggestion
title: TRG 1.09 - UX consistency / Styleguide for UI
```

File: docs/release/trg-0/trg-1-09.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status                                    | Created        | Post-History                                       |
+|-------------------------------------------|-----------------|----------------------------------------------------|
+| Draft - MISSING CLOSED INFORMATION      | 10-Jan-2024     | Transfered from Release Acceptance Criteria into TRG|
```
Comment (FaGru3n) [inline docs/release/trg-0/trg-1-09.md:12]:
```suggestion
| Draft | 10-Jan-2024     | Transfered from Release Acceptance Criteria into TRG|
```

File: docs/release/trg-0/trg-1-09.md (hunk 3 above)
Comment (FaGru3n) [inline docs/release/trg-0/trg-1-09.md:21]:
```suggestion
- User Interfaces **should** align with the Tractus-X Style Guidelines.
```

**Review by FaGru3n — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
@kelaja @RolaH1t & @danielmiehle what about merging this so that I can create my PR with the new content?

**Comment by RolaH1t** [comment]
> @kelaja @RolaH1t & @danielmiehle what about merging this so that I can create my PR with the new content?

ok from my side

**Comment by kelaja** [comment]
@jjeroch @FaGru3n there seems to be a check failing which prevents merging. Kindly guide how this can be fixed.

**Comment by FaGru3n** [comment]
> @jjeroch @FaGru3n there seems to be a check failing which prevents merging. Kindly guide how this can be fixed.

https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-03#checking-contributions-by-a-eclipse-contributor

**Comment by Siegfriedk** [comment]
Introducing this TRG in my branch due to @kelaja being away for a bit and im unable to merge this PR due to a conflict @kelaja would need to fix.

**Pull request closed without merging**